quick_cluster_install/ansible-playbooks/bootstrap-env-playbook.yml

```diff
@@ -0,0 +1,52 @@
+- hosts: 127.0.0.1
+  connection: local
+
+  vars:
+    install_config:
+      - '{{env_name}}'
+      - '{{site_host}}'
+      - '{{server_host_name}}'
+      - '{{ssh_public_key}}'
+      - '{{ssh_username}}'
+      - '{{cchq_venv}}'
+    config_is_not_valid: '{{"" in install_config}}'
+    envs_path: "{{ ansible_env.HOME }}/environments"
```

Comment:
Is there a way we can make use of https://github.com/dimagi/commcare-cloud/blob/master/quick_monolith_install/bootstrap-env-playbook.yml to make this DRYer?

Reply:
Ah, I read the description just a little latter. So you can disregard this, unless if you really want to try that.